AlfrescoDocumentPickerFileProvider/AFPDataManager.m

```diff
@@ -209,6 +220,20 @@ - (void)updateMetadataForIdentifier:(NSFileProviderItemIdentifier)itemIdentifier
     }
 }
 
+- (AFPItemMetadata *)localFilesItem
+{
+    AFPItemMetadata *localFiles = nil;
+    RLMRealm *realm = [self realm];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"identifier = %@", [AFPItemIdentifier itemIdentifierForSuffix:nil andAccountIdentifier:nil]];
```

Comment:
even though both forms of matching within predicates are accepted "=" and "==" for readability I would recommend sticking to "==" since this type of matching is widely used